src/rng.rs

```diff
@@ -213,12 +210,8 @@ mod test {
         }
     }
 
-    impl Rand for Seed {
-        fn rand<R: Rng>(rng: &mut R) -> Self {
-            // Note: the `wrapping_add` trick is a workaround for what seems to be a weakness in
-            // `XorShiftRng`. Without it we would sometimes end up with multiple rngs producing
-            // identical values.
-            // The idea is taken from: https://github.com/maidsafe/maidsafe_utilities/blob/24dfcbc6ee07a14bf64f3bc573f68cea01e06862/src/seeded_rng.rs#L92
+    impl Distribution<Seed> for Standard {
+        fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Seed {
             Self([
                 rng.next_u32().wrapping_add(rng.next_u32()),
```

Comment:
If the comment goes, shouldn't the trick go too?
But since we are still using XorShift, shouldn't the comment stay?

Reply:
Ah yes it should, I agree, good catch